tests/testsuite/jobserver.rs

```diff
@@ -176,7 +177,7 @@ fn jobserver_and_j() {
     } else {
         "make"
     };
-    if Command::new(make).arg("--version").output().is_err() {
+    if !is_ci() && Command::new(make).arg("--version").output().is_err() {
```

Comment:
Is there any technical reason only running this on CI? I can see that support `mingw32-make` in `requires_*` is a bit annoying. 

Maybe we can add another rule `ci_only` to explicit annotate these kind of tests as `#[ignore]`?

Reply:
I'm not sure I'm understanding.  This test runs in CI or a developer environment that has `make` installed.  It is only ignored in developer environments that are missing `make`.

This change is to force it to run in CI, because we want to make sure they run there.

And yea, it's a bit awkward to place this requirement in the test itself. I did some checking, and it looks like it should be fine to just use "make" on Windows. This wasn't discussed in the original PR, but I suspect it was some issue with how the Appveyor environment was set up.

Reply:
Oh my fault. I understood it wrong.